docs/docs/identity-providers/gitlab.md

```diff
@@ -25,8 +25,19 @@ Field        | Description
 ------------ | --------------------------------------------
 Name         | The name of your web app
 Redirect URI | `https://${authenticate_service_url}/oauth2/callback`
-Scopes       | **Must** select **read_user** and **openid**
+Scopes       | **Must** select **openid**, **read_user** and **api**
 
+If no scopes are set, we will use the following scopes:
```

Comment:
The formatting of this list is a bit off. See : https://deploy-preview-614--pomerium-docs.netlify.app/docs/identity-providers/gitlab.html

What do you think about listing them in line like....
If no scopes are set, we will use the following scopes: `openid`, `api`, `read_user`,`profile`, and `email`.